ironfish-rust/src/transaction/mints.rs

```diff
@@ -147,6 +147,11 @@ impl UnsignedMintDescription {
         Ok(self.description)
     }
 
+    pub fn add_signature(mut self, signature: Signature) -> Result<MintDescription, IronfishError> {
+        self.description.authorizing_signature = signature;
+        Ok(self.description)
+    }
+
```

Comment:
We don't need the result type, can just be a plain `MintDescription` return type

Reply:
Done